### PR TITLE
EDP utility

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- tools to interact with EDP

**Related issue(s)**

### CODE REVIEW

**Review by jaroslaw-pieszka — APPROVED** [review]

File: utils/edp-registrator/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+### Running
+
+#### Setting environment variables
+
+Before using `edp` tool you must set environment variables:
+
+1. Copy existing template file, for example: 
+`cp env.dev.template env.dev`
+2. Set missing secret value in the file
+3. Export environment variables:
```
Comment (IwonaLanger) [inline utils/edp-registrator/README.md:31]:
```suggestion
3. Export the environment variables:
```

File: utils/edp-registrator/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+`export $(grep -v '^#' env.dev | xargs)`
+
+#### Running a command
```
Comment (IwonaLanger) [inline utils/edp-registrator/README.md:34]:
```suggestion
#### Run a Command
```

File: utils/edp-registrator/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Get metadata from EDP:
```
Comment (IwonaLanger) [inline utils/edp-registrator/README.md:36]:
```suggestion
Get metadata from the EDP:
```

File: utils/edp-registrator/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+```shell
+./edp get <subaccountID>
+```
+
+Register subaccount in EDP:
```
Comment (IwonaLanger) [inline utils/edp-registrator/README.md:41]:
```suggestion
Register a subaccount in the EDP:
```

File: utils/edp-registrator/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+```shell
+./edp register <subaccount ID> <platform region> <plan>
+```
+for example:
+```shell
+./edp register 41ba3cf2-041d-4223-adfe-c5de3458acbe cf-us21 standard
+```
+
+Deregister subaccount from EDP:
```
Comment (IwonaLanger) [inline utils/edp-registrator/README.md:50]:
```suggestion
Deregister a subaccount from the EDP:
```

File: utils/edp-registrator/README.md (hunk 2 above)
Comment (IwonaLanger) [reply utils/edp-registrator/README.md:34]: Please consider using bullet points in this section.

**Review by IwonaLanger — APPROVED** [review]

File: utils/edp-registrator/checksubaccounts.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+notExistingSb = []
+for item in f.readlines():
+    item = item.strip(" \n")
+    id = item.split(" ")[0]
+    if id not in edpSubaccounts:
+        print(item)
+        count = count + 1
+        notExistingSb.append("'"+id+"'")
+f.close()
+print(count)
```
Comment (jaroslaw-pieszka) [inline utils/edp-registrator/checksubaccounts.py:30]: We could use len(notExistingSb) and count wouldn't be needed.

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**